src/main/java/practice/StreamPractice.java

```diff
@@ -75,6 +108,10 @@ public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
      * parametrized with Candidate in CandidateValidator.
      */
     public List<String> validateCandidates(List<Candidate> candidates) {
-        return Collections.emptyList();
+        return candidates.stream()
+                .sorted(Comparator.comparing(Candidate::getName))
+                .filter(new CandidateValidator())
```

Comment:
It`s not a mistake, but it is better to remove candidates from stream before sorting, so that sort will work faster with smaller amount of candidates.
```suggestion
                .filter(new CandidateValidator())
                .sorted(Comparator.comparing(Candidate::getName))
```